recipes/libradtran/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "rubin-libradtran" %}
+{% set version = "2.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://www.libradtran.org/download/libRadtran-{{ version }}.tar.gz
+  sha256: 64db9636aea2adaae46bac86dfcc0c72b670edf71d600558a1e5f8ad4e88ab6b
+  patches:
+    - 0001-support-python3.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - make
+    - autoconf
+    - gnuconfig
+    - automake
+  host:
+    - flex
+    - perl
+    - libnetcdf
+    - netcdf4
+    - gsl
+    - python
+  run:
+    - python
+    - gsl
+    - libnetcdf
+    - netcdf4
+    - libopenblas
+    - jpeg
+    - hdf5
```

Comment:
Please double check the host vs run lists here. Many of these packages have run exports and so do not need to be in run. 

Also, IIRC the blas pinning does not follow our docs.

Reply:
Okay, I think I've fixed the blas pinning, and I've simplified the run exports hopefully not too far.  It took some trial and error to get to the point where it did run, but I guess not in the optimal configuration.

Reply:
right ideally our linter would let you know if a package doesn't have run exports. In practice you have to go look at the feedstock to know.